scripts/add-local-ip-certs-to-docker-4.x.sh

```diff
@@ -36,6 +36,14 @@
 # ./add-local-ip-certs-to-docker-4.x.sh cht-3-14 expire
 #
 
+update_nginx_local_ip_tls_cert(){
+  nginxContainerId=$1
+  curl --fail --silent --show-error -o /tmp/local-ip-fullchain https://local-ip.medicmobile.org/fullchain
+  curl --fail --silent --show-error -o /tmp/local-ip-key https://local-ip.medicmobile.org/key
```

Comment:
What do you think of making `curl` retry the requests to cover for the occasional networking issue?

https://stackoverflow.com/questions/42873285/curl-retry-mechanism/42873372#42873372

```suggestion
  curl --retry 3 --fail --silent --show-error -o /tmp/local-ip-fullchain https://local-ip.medicmobile.org/fullchain
  curl --retry 3 --fail --silent --show-error -o /tmp/local-ip-key https://local-ip.medicmobile.org/key
```

Reply:
wow - excellent thinking!  Is there anything curl can't do?!?! [TIL](https://everything.curl.dev/usingcurl/downloads/retry.html)!